capi/VideoDecoderCapi.h

```diff
@@ -20,8 +20,8 @@
 #ifndef __ENABLE_CAPI__
 #define __ENABLE_CAPI__ 1
 #endif
-#include "interface/VideoDecoderDefs.h"
-#include "interface/VideoCommonDefs.h"
+#include "VideoDecoderDefs.h"
+#include "VideoCommonDefs.h"
```

Comment:
We have installed a copy of VideoDecoderCapi.h to $prefix/include/libyami/ and yami-utils never use it. it use the local one. ;-)